docs/installation/presto-cli-installation.rst

```diff
@@ -9,18 +9,17 @@ The Presto CLI provides a terminal-based interactive shell for running queries.
 Download the `presto-cli` and copy it to the location you want to run it from. This location may be any node that has network access to the coordinator. Then rename it to presto and make it executable with chmod +x:
 ::
 
- $ mv presto-cli-0.115-executable.jar presto
+ $ mv presto-cli-*-executable.jar presto
```

Comment:
oh haha, I've gotten caught by the angle bracket thing before too.  I guess github treats them as some kind of tag.   Seems like we're saying the same thing then